utility/SignalProbe.cpp

```diff
@@ -158,6 +163,30 @@ class SignalProbe : public Pothos::Block
             mean /= N;
             _value = mean;
         }
+        else if (_mode == "MIN")
+        {
+            double minimum = std::numeric_limits<double>::max();
+            ProbeType x_n;
+            for (size_t n = 0; n < N; n++)
+            {
+                x_n = Pothos::Util::fromQ<ProbeType>(x[n], 0);
+                const double v = std::abs(x_n);
```

Comment:
was the abs() intended?

Reply:
This calculates minimum or maximum magnitudes only, and that's what I implemented and documented.

For minimum/maximum value of real numeric types, I could think about implementing separate `MINVAL`/`MAXVAL` modes and renaming `MIN`/`MAX` to `MINMAG`/`MAXMAG`. However I haven't looked into how to deal with the templated types for that yet. Thoughts?

Reply:
For some additional context, I wanted to feed the output of an FFT to this probe and use `MAX` to find the peak value.

Reply:
> For minimum/maximum value of real numeric types, I could think about implementing separate MINVAL/MAXVAL modes and renaming MIN/MAX to MINMAG/MAXMAG. However I haven't looked into how to deal with the templated types for that yet. Thoughts?

I like that idea. 

I dont think the different templated instantiations should give you any trouble. Its just a couple more if/else clauses without the abs.

Reply:
The issue was dealing with min/max for complex types, which isn't defined. I added templated functions to use `abs` for complex types.